src/checkpoint/checkpoint_txn.c

```diff
@@ -722,6 +723,11 @@ __checkpoint_stats(WT_SESSION_IMPL *session)
     /* Compute timer statistics for the checkpoint prepare. */
     msec = WT_TIMEDIFF_MS(conn->ckpt.prepare.timer_end, conn->ckpt.prepare.timer_start);
     __checkpoint_timer_stats_set(&conn->ckpt.prepare, msec);
+
+    /* update reuse ratio stats. */
+    WT_STAT_CONN_SET(session, block_reusable_over_50, 0);
+    WT_STAT_CONN_SET(session, block_reusable_over_90, 0);
+    WT_IGNORE_RET(__wt_conn_btree_apply(session, NULL, __checkpoint_block_stats, NULL, NULL));
```

Comment:
Even if it always returns 0, it could change and it would be best to already have the right macro. Note that i have changed the function name.
```suggestion
    WT_RET(__wt_conn_btree_apply(session, NULL, __checkpoint_block_reusable_stats, NULL, NULL));
```

Reply:
The point here is this function is `void` return type.

Reply:
@Sean04 - what are your thoughts on this? I am not a fan of ignoring potential errors from `__wt_conn_btree_apply`.

Reply:
I think we should change the return type of the function

Reply:
Sounds good, please proceed with this @whoisong